### DISCUSSION TITLE
Error: error - update album fail

### DISCUSSION THREAD

**Discussion by vuongquoc** [LycheeOrg/Lychee#2266] [Q&A] [answered]
Hi,
After get V5 to work. 
i can create new albums normally until got fail upload image and then can not create new album too...
i checked in error log and found below error every time i try to make new album...
Do any one have the same problem? and how can i fix it?

Thanks.
-----
PDOStatement::execute:580 SQLSTATE: Numeric value out of range: 1264 Out of range value for column 'legacy_id' at row 1
App\Models\BaseAlbumImpl::performInsert:105 SQLSTATE: Numeric value out of range: 1264 Out of range value for column 'legacy_id' at row 1; caused by
Illuminate\Database\Connection::runQueryCallback:822 SQLSTATE: Numeric value out of range: 1264 Out of range value for column 'legacy_id' at row 1 (Connection: mysql, SQL: insert into `base_albums` (`id`, `legacy_id`, `created_at`, `updated_at`, `title`, `description`, `owner_id`, `sorting_col`, `sorting_order`, `is_nsfw`) values (qg89YWaFPYhWNnZiXnqygWQ3, 17066129157400, 2024-01-30 11:08:35.739626, 2024-01-30 11:08:35.739626, 2018, ?, 1, ?, ?, 0)); caused by
App\Models\BaseAlbumImpl::performInsert:105 SQLSTATE: Numeric value out of range: 1264 Out of range value for column 'legacy_id' at row 1 (Connection: mysql, SQL: insert into `base_albums` (`id`, `legacy_id`, `created_at`, `updated_at`, `title`, `description`, `owner_id`, `sorting_col`, `sorting_order`, `is_nsfw`) values (qg89YWaFPYhWNnZiXnqygWQ3, 17066129157400, 2024-01-30 11:08:35.739626, 2024-01-30 11:08:35.739626, 2018, ?, 1, ?, ?, 0)); caused by
App\Exceptions\ModelDBException::create:50 Updating base album impl failed; caused by
App\Exceptions\ModelDBException::create:50 Updating album failed; caused by
Context:

[
  [
    22003
  ],
  [
    22003
  ],
  [
    22003
  ],
  [
    22003
  ]
]

**Comment by ildyria** [comment, marked as answer]
Go into your settings => More at the bottom of the page, find `force_32bit_ids` set it to `1` and save.

**Reply by vuongquoc** [reply]
Thank you, @ildyria 
It fixed!!!

VuongQuoc
